test/modules/core/effects/lighting/sunlight.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable */
+import test from 'tape-catch';
+import {getSolarPosition} from '@deck.gl/core/effects/lighting/sunlight';
+
+const MS_A_HOUR = 3.6e6;
+
+function almostEqual(v1, v2, epsilon = 0.3) {
```

Comment:
Nit: 
* 0.3 sounds like a big epsilon (epsilon is normally like 10e-10 etc), maybe comment on what units these are. 
* As an option, math.gl has an equals method with configurable epsilon.